app/services/LiabilityCalculationService.scala

```diff
@@ -16,17 +16,24 @@
 
 package services
 
-import connectors.LiabilityCalculationConnector
 import connectors.httpParsers.LiabilityCalculationHttpParser.LiabilityCalculationResponse
-import javax.inject.Inject
+import connectors.{LiabilityCalculationConnector, PostCalculateIncomeTaxLiabilityConnector}
 import uk.gov.hmrc.http.HeaderCarrier
 
+import javax.inject.Inject
 import scala.concurrent.Future
 
-class LiabilityCalculationService @Inject()(liabilityCalculationConnector: LiabilityCalculationConnector) {
+class LiabilityCalculationService @Inject()(liabilityCalculationConnector: LiabilityCalculationConnector,
+                                            postCalculateIncomeTaxLiabilityConnector: PostCalculateIncomeTaxLiabilityConnector) {
 
-  def calculateLiability(nino: String, taxYear: String, crystallise: Boolean)(implicit hc: HeaderCarrier): Future[LiabilityCalculationResponse] = {
-    liabilityCalculationConnector.calculateLiability(nino, taxYear, crystallise)
+  def calculateLiability(nino: String, taxYear: String, crystallise: Boolean)
+                        (implicit hc: HeaderCarrier): Future[LiabilityCalculationResponse] = {
+    if (taxYear.equals("2024")) {
```

Comment:
we are planning to use another API for the taxYear > 2024, so it seem that pattern matching would be intuitive choice here as more changes is expected